guacamole-docker/bin/start.sh

```diff
@@ -710,6 +710,17 @@ associate_json() {
     ln -s /opt/guacamole/json/guacamole-auth-*.jar "$GUACAMOLE_EXT"
 }
 
+##
+## Adds properties to guacamole.process wich configure other
+## options
```

Comment:
I think "other options" is a bit too vague here. It's unclear from reading the documentation for `associate_other()` exactly which configuration properties are in scope of the function. If the idea is that `associate_other()` covers any properties that are not tied to a specific extension, then I think that should be specified here.

Alternatively, there is already a section devoted to the few options that do not relate to an extension:

https://github.com/apache/guacamole-client/blob/c143c7cb5c73bcae51a4576c3bbe7d1d76ef7620/guacamole-docker/bin/start.sh#L794-L796

Simply placing the relevant call to `set_optional_property()` there may be better and simpler.